### PR TITLE
Api 34936 ptcpnt addrs create

### PR DESCRIPTION
**Pull request by mchristiansonVA**
## Summary

- Adds the VnpPtcpntAddrsService with associated test and VCR cassette.

## Related issue(s)

[API-34936](https://jira.devops.va.gov/browse/API-34936)

## Testing done

- [x] *New code is covered by unit tests*
- Tested via rspec test and VCR cassette creation.

## What areas of the site does it impact?
*(Describe what parts of the site are impacted and*if*code touched other areas)*

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [ ]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [ ]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

### CODE REVIEW

**Comment by stevenjcumming** [comment]
@mchristiansonVA please ask a teammate for a review then I'll review it

File: modules/claims_api/spec/lib/claims_api/vnp_ptcpnt_addrs_service_spec.rb
Comment (FonzMP) [inline modules/claims_api/spec/lib/claims_api/vnp_ptcpnt_addrs_service_spec.rb:54]: we may want to merge and use the new recording setup to keep in line with our new BGS service specs. it's also semi-dependent on [these changes](https://github.com/department-of-veterans-affairs/vets-api/pull/16232), so could wait for that to merge before doing so.

**Review by ericboehs — APPROVED** [review]

File: modules/claims_api/spec/lib/claims_api/vnp_ptcpnt_addrs_service_spec.rb
Comment (nihil2501) [reply modules/claims_api/spec/lib/claims_api/vnp_ptcpnt_addrs_service_spec.rb:54]: Good point. I'll ask someone for a brief hands-on safety check with that big PR at the beginning of tomorrow so we feel good about merging it. The PR already has the necessary approvals and passes all checks so we can get it merged in quickly. Then we can slightly restructure this one to match the new BGS unit test pattern which I'm free to help demo at any time if you would like @mchristiansonVA or you can copy off of one of the two new `ManageRepresentativeService` specs.

**Review by nihil2501 — APPROVED** [review]
LGTM

**Review by nihil2501 — APPROVED** [review]
LGTM

**Pull request merged**
